compliance/collection/auditlog/auditlog_impl.go

```diff
@@ -34,16 +33,12 @@ var (
 
 type auditLogReaderImpl struct {
 	logPath    string
-	stopC      concurrency.Signal
+	stopper    concurrency.Stopper
 	sender     auditLogSender
 	startState *storage.AuditLogFileState
 }
 
 func (s *auditLogReaderImpl) StartReader(ctx context.Context) (bool, error) {
-	if s.stopC.IsDone() {
-		return false, errors.New("Cannot start reader because stopC is already done. Reader might have already been stopped")
-	}
```

Comment:
This was redundant because nothing in the current code tries to stop the reader before it's started.